app/views/course/assessment/submission/submissions/_question.json.jbuilder

```diff
@@ -17,4 +17,14 @@ json.type case question.actable_type
             'Scribing'
           end
 
+# In order to be classified as a File Submission question
+# 	1. The question must be a Java question
+# 	2. There can be either no submission files or
+# 	3. The files must be .java files
+fileSubmission = answer.is_a?(Course::Assessment::Answer::Programming) &&
```

Comment:
@zhengwei143 I remember that there's an option indicates the file submission question, can we use that one instead ? Otherwise we need to explicitly store the option somewhere, because the logic here is over complex and will be probably missed after a few refactoring.